CHANGES/README.rst

```diff
@@ -1,15 +1,23 @@
-.. _Adding change notes with your PRs:
+.. _Making a pull request:
+
+Making a pull request
+=====================
+
+When making a pull request, please include a short summary of the changes
+and a reference to any issue tickets that the PR is intended to solve.
+All PRs with code changes should include tests. All changes should
```

Comment:
Yeah, I missed that this now refers to tests that are not related to the https://github.com/aio-libs/aiohttp/tree/master/CHANGES#readme directory at all.